packages/paste-core/components/alert/src/index.tsx

```diff
@@ -84,11 +84,14 @@ const Alert: React.FC<AlertProps> = ({children, onDismiss, variant, role, ...pro
     </Box>
   );
 };
-Alert.propTypes = {
-  children: PropTypes.node.isRequired,
-  onDismiss: PropTypes.func,
-  role: PropTypes.string,
-  variant: PropTypes.oneOf(Object.keys(AlertVariants).map(variant => AlertVariants[variant])).isRequired,
-};
 Alert.displayName = 'Alert';
+
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Just had a thought, does this get stripped out on our production build?

Reply:
I had the same thought last night, and yup it does :(.  I'll just remove it for now and we should encourage folks to use the babel transform

Reply:
🎉